### PR TITLE
feat : added Note app Screens

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Added home page , add new note page , trash and archive page 

Link preview : https://deploy-preview-1--notify-note-app.netlify.app/

### CODE REVIEW

**Review by RakshithBhat03 — APPROVED** [review]
- The code looks clean and well structured.
- Good component folder structure.
- Good use of css variables.

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,25 @@
-import "./App.css";
-import logo from "./logo.png";
+import "./styles.css";
+import { Routes, Route } from "react-router-dom";
+import Header from "./component/header";
```
Comment (Kothalkarpriya) [inline src/App.js:3]: You can import all the components from a single file created as an index and import all the components at once and export from their, and you can save your space in other files

File: src/styles.css [hunk 2]
```diff
@@ -1,3 +1,16 @@
+@import "https://getratha.netlify.app/styles.css";
+@import "./style/sidenavbar.css";
+@import "./style/card.css";
```
Comment (Kothalkarpriya) [inline src/styles.css:3]: for these files also you can import and export from a single file

**Review by Kothalkarpriya — APPROVED** [review]
Everything looks good, just added some suggestions

**Pull request merged**